src/libasr/runtime/lfortran_intrinsics.c

```diff
@@ -2939,6 +2938,41 @@ LFORTRAN_API char *_lpython_get_argv(int32_t index) {
     return _argv[index];
 }
 
+
+LFORTRAN_API char *_lfortran_get_command_command() {
+    char* out;
+    for(int i=0; i<_argc; i++) {
+        if(i == 0) {
+            out = strdup(_argv[i]);
+        } else {
+            out = realloc(out, strlen(out) + strlen(_argv[i]) + 1);
+            strcat(out, " ");
+            strcat(out, _argv[i]);
+        }
+    }
+    return out;
+}
+
+
+LFORTRAN_API int32_t _lfortran_get_command_length() {
+    char* out;
+    for(int i=0; i<_argc; i++) {
+        if(i == 0) {
+            out = strdup(_argv[i]);
+        } else {
+            out = realloc(out, strlen(out) + strlen(_argv[i]) + 1);
+            strcat(out, " ");
+            strcat(out, _argv[i]);
+        }
+    }
```

Comment:
```suggestion
    char* out = _lfortran_get_command_command();
```

I think you'll be able to reuse it.